makeflow/src/makeflow.c

```diff
@@ -144,6 +144,11 @@ static void makeflow_wrapper_add_command( const char *cmd )
 	}
 }
 
+/* XXX 
+ * makeflow_wrapper_add_input_file() and
+ * makeflow_wrapper_add_output_file() do not work.
+ * replace "optarg" by "file" ?
+ */
```

Comment:
Ah, there is a bug in `wrapper_add_output_file`, it should be manipulating `wrapper_input_files` instead of `wrapper_output_files`, you can fix that.